src/App.tsx

```diff
@@ -362,280 +287,242 @@ const formatPercentage = (a: number, b: number): string =>
     const quotient = b === 0 ? 0 : a / b
     return `${(Math.round(quotient * 10000) / 100).toFixed(2)}%`
   }
-const getPartyById = (id: string) =>
-  election.parties.find((party) => party.id === id)
-const sumCandidateVotes = (candidates: ResultsCandidates): number =>
-  Object.keys(candidates).reduce((sum, key) => sum + candidates[key], 0)
-const datesAreOnSameDay = (first: Date, second: Date): boolean =>
-  first.getFullYear() === second.getFullYear() &&
-  first.getMonth() === second.getMonth() &&
-  first.getDate() === second.getDate();
-
-// pre-election || during-election || post-election
+
+const sumCompressedTallies = (compressedTallies: CompressedTally[]): CompressedTally =>
+  compressedTallies.reduce(
+    (sum, tally) => sum.length === 0
+      ? tally
+      : sum.map(
+        (contest, contestIndex) => contest.map(
+          (option, tallyIndex) => option + tally[contestIndex][tallyIndex]
+        )
+      ),
+    []
+  )
+
 const refreshInterval = 60
 const App: React.FC = () => {
+  const electionHash = process.env.REACT_APP_ELECTION_HASH
 
-  const getElectionStatus = () => {
-    const now = new Date()
-    if (now <= new Date(election.polls.openAt)) {
-      return 'pre-election'
-    }
-    if (now >= new Date(election.polls.closeAt)) {
-      return 'post-election'
-    }
-    return 'during-election'
-  }
-  const isElectionDay = datesAreOnSameDay(new Date(), new Date(election.date))
-  const [ electionStatus, setElectionStatus ] = useState(getElectionStatus)
-  const [ results, setResults ] = useState<Results | undefined>(undefined)
+  const [ election, setElection ] = useState<Election | undefined>(undefined)
+  const [ tallies, setTallies ] = useState<ServerResult[] | undefined>(undefined)
   const [ refreshCountdown, setRefreshCountdown ] = useState(0)
 
-  const hasResults = !!results?.ballotsCounted
-  const [ currentPage, setCurrentPage ] = useState(hasResults ? 'results' : 'info')
+  const hasResults = !!tallies?.length
+  const [ currentPage, setCurrentPage ] = useState('results')
 
-  const fetchResults = async () => {
-    const response = await fetch(process.env.REACT_APP_API || "https://err-backend-worker.votingworks.workers.dev/warren")
+  const fetchElection = async () => {
+    const response = await fetch(`https://results.voting.works/election/${process.env.REACT_APP_ELECTION_HASH}/definition`)
```

Comment:
worth URL-encoding the election hash before stuffing it in the URL, even if it's not supposed to have any weird characters, just in case a misconfiguration causes this to do some weird fetching that breaks later.

Reply:
done